docat/tests/test_index.py

```diff
@@ -406,8 +406,8 @@ def test_index_all_projects_creates_file_and_version_index_api(client_with_claim
 
         for project in projects:
             for version in versions:
-                mock_insert_version_into_version_index.assert_any_call(docat.DOCAT_INDEX_PATH, project, version, [])
-                mock_insert_file_index_into_db.assert_any_call(docat.DOCAT_INDEX_PATH, project, version, "index.html", "hello world")
+                mock_insert_version_into_version_index.assert_any_call(ANY, project, version, [])
```

Comment:
Why do you use any here?

Reply:
Because I changed the parameter from the index db path to the actual db instance. I then passed in `docat.index_db` but the test failed all the time because it thought the two instances were different somehow

Reply:
Ok, I fixed it and wrote a test for it.